doc/efuns/arrays/allocate.3

```diff
@@ -5,12 +5,19 @@
 allocate() - allocate an array
 
 .SH SYNOPSIS
-mixed *allocate( int size );
+varargs mixed *allocate( int size, mixed value );
 
 .SH DESCRIPTION
 Allocate an array of <size> elements.  The number of elements must be >= 0
 and not bigger than a system maximum (usually ~10000).  All elements are
-initialized to 0.
+initialized to 0 by default.
+
+If the optional second argument is provided, the values are intialized to
```

Comment:
It is currently a bit unclear what the accepted types for the second argument are. I assume it's a mixed array | function but the wording could be a bit clearer.

Reply:
https://github.com/fluffos/fluffos/blob/master/src/packages/core/core.spec 

it should match here